specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,39 @@
+# Commitment Discount Type
+
+A commitment-based discount is a contractual commitment in exchange for a discounted unit price. Commitment discounts are available as spend-based or resource-based discounts, depending on the CSP offering them and the particular service.
```

Comment:
```suggestion
A commitment-based discount is a contractual commitment in exchange for a discounted unit price. Commitment discounts are available as spend-based or resource-based discounts, depending on the Provider's offering.
```

Reply:
Also, not a fan of resource-based 'discounts'. Its really a discount due to a commitment for an amount of spend or resource consumption. 

FinOps documentation refers to this as resource-based commitments vs spend-based commitments.

Reply:
```suggestion
A commitment-based discount is a contractual commitment in exchange for a discounted unit price. These discounts may be available for spend-based commitments or resource-based commitments, depending on the Provider's offerings.
```

Reply:
Can we move these comments to the CommitmentDiscountId PR? Given that's further along, I'd like to close the wording there. This PR should just copy what it has when finalized.

@rodneyjoyce along the same lines, if there's anything you feel is missing there, can you add a comment as well?